app/components/subcases/subcase-header.js

```diff
@@ -193,13 +201,63 @@ export default class SubcasesSubcaseHeaderComponent extends Component {
 
   moveSubcase = task(async () => {
     const oldDecisionmakingFlow = await this.args.subcase.decisionmakingFlow;
+
+    const oldCase = await oldDecisionmakingFlow.case;
+    const newCase = await this.newDecisionmakingFlow.case;
+
+    const parliamentRetrievalActivity = await this.args.subcase.parliamentRetrievalActivity;
+
+    for (const submission of this.submissions) {
+      submission.decisionmakingFlow = this.newDecisionmakingFlow;
+      await submission.save();
+    }
+
+    if (parliamentRetrievalActivity) {
+      const newParliamentFlow = await newCase.parliamentFlow;
+      if (newParliamentFlow) {
+        const oldParliamentSubcase = await parliamentRetrievalActivity.parliamentSubcase;
+        const oldParliamentFlow = await oldParliamentSubcase.parliamentFlow;
+
```

Comment:
we only come in here when there is a retrieval.
Then we check if the new case has a parliament flow already.
What about submission activities?
Do we need to update VP for that? (when there is a parliament flow but no retrieval yet)

Reply:
nvm, those submissions are not attached to subcases directly